### DISCUSSION TITLE
Nuclei executes without any action

### DISCUSSION THREAD

**Discussion by ManasHarsh** [projectdiscovery/nuclei#467] [Q&A] [answered]
Hi team,

I have been using nuclei since a while but never faced this issue. Nuclei executes so fast and doesn't check the templates. This tool is not new to me but don't know this is not working. Any solution?

**Comment by geeknik** [comment]
So that we may better help you diagnose this issue, can you share with us the version of nuclei you are using (`nuclei -version`) along with your command line?

**Comment by ehsandeep** [comment, marked as answer]
@manasjha7 A most common reason for this would be passing the list of subdomains instead of URLs, otherwise, you can simply check this on your side with `-v` or `-debug` flag and see what's going on, as geeknik pointed out, if you can share more details, we can definitely help to identify the issue.
